Description: add so version to duckdb library
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 duckdb (0.10.0-22) jammy; urgency=medium
 .
   * amend man
Author: Elliana May <me@mause.me>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2024-02-29

--- duckdb-0.10.0.orig/src/CMakeLists.txt
+++ duckdb-0.10.0/src/CMakeLists.txt
@@ -38,6 +38,7 @@ if(AMALGAMATION_BUILD)
 
   add_library(duckdb SHARED "${PROJECT_SOURCE_DIR}/src/amalgamation/duckdb.cpp")
   target_link_libraries(duckdb ${DUCKDB_SYSTEM_LIBS})
+  set_target_properties(duckdb PROPERTIES VERSION ${DUCKDB_NORMALIZED_VERSION} SOVERSION ${DUCKDB_VERSION_MAJOR})
   link_threads(duckdb)
   link_extension_libraries(duckdb)
 
